api/feedback_test.go

```diff
@@ -30,23 +17,44 @@ var (
 		"name": "Mr Feedback reporter",
 		"email_address": "feedback@reporter.com"
 	}`
-	isPageUseful      = true
-	isGeneralFeedback = false
+	isPageUseful       = true
+	isGeneralFeedback  = true
+	notGeneralFeedBack = false
 )
 
 var expectedEmail = `From: sender@mail.com
 To: receiver@mail.com
 Subject: Feedback received
 
-Is page useful: true
-Is general feedback: false
+Feedback Type: A specific page
+Page URL: https://testhost:1234/sub/path
```

Comment:
Should we be printing the URL if it's general feedback? Maybe these should be mutually exclusive?